packages/yuudachi/src/Constants.ts

```diff
@@ -42,6 +42,15 @@ export const EMBED_FIELD_LIMIT = 25;
 export const EMBED_FIELD_NAME_LIMIT = 256;
 export const EMBED_FIELD_VALUE_LIMIT = 1024;
 export const SNOWFLAKE_MIN_LENGTH = 17;
-export const AUTOMOD_FLAG_INDICATOR_FIELD_NAME = 'flagged_message_id';
 export const AUTOCOMPLETE_CHOICE_LIMIT = 25;
 export const AUTOCOMPLETE_CHOICE_NAME_LENGTH_LIMIT = 100;
+
+export const TRUST_AND_SAFETY_URL_BASE = 'https://support.discord.com/hc/en-us/requests/new';
+
+export const DiscordLocales: { [key: string]: string } = {
+	'en-US': 'en-us',
+	'en-GB': 'en-gb',
+	'es-ES': 'es',
+	'zh-CN': 'zh-tw',
+	'zh-TW': 'zh-tw',
+};
```

Comment:
Whats up with this list?

Reply:
Discord locales does not always match Yuuki ones, so this table is for conversion with the Trust and Safety url as mentioned by will (https://github.com/Naval-Base/yuudachi/pull/1103#discussion_r947353938) but this could be optimized ngl

Reply:
Improved in 78a6dc8